examples/example-uvtm.c

```diff
@@ -0,0 +1,42 @@
+
+// A libuv timeout example
+// by github dot com slash michael-grunder
+// many thanks!
+
+// usage: make hiredis-example-uvtm && ./examples/hiredis-example-uvtm
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <signal.h>
+
+#include <hiredis.h>
+#include <async.h>
+#include <adapters/libuv.h>
+#include <time.h>
+#include <sys/time.h>
+
+void sleepCb(redisAsyncContext *c, void *r, void *privdata) {
+    printf("sleep CB\n");
```

Comment:
When the async command timeout the reply should be `NULL`. This can be shown in this example by using
```
    redisReply *reply = r;
    assert(reply == NULL);
```
or by some simple printf.